controllers/cloudstorages/aws_test.go

```diff
@@ -0,0 +1,153 @@
+package cloudstorages
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("GetName", func() {
+	var (
+		provider      string
+		expectedValue string
+	)
+	BeforeEach(func() {
+		provider = "aws"
+		expectedValue = "aws"
+	})
+	Context("Provider is aws", func() {
+		It("should get provider name = aws", func() {
+			csp := &AWSCloudStorageProvider{providerName: provider}
+			Expect(csp.GetName()).To(Equal(expectedValue))
+		})
+	})
+})
+
+var _ = Describe("GetCloudStoragePath", func() {
+	var (
+		provider      string
+		bucket        string
+		gatlingName   string
+		subDir        string
+		expectedValue string
+	)
+	BeforeEach(func() {
+		provider = "aws"
+		bucket = "testBucket"
+		gatlingName = "testGatling"
+		subDir = "subDir"
+		expectedValue = "s3:testBucket/testGatling/subDir"
+	})
+	Context("Provider is aws", func() {
+		It("path is aws s3 bucket", func() {
+			csp := &AWSCloudStorageProvider{providerName: provider}
+			Expect(csp.GetCloudStoragePath(bucket, gatlingName, subDir)).To(Equal(expectedValue))
+		})
+	})
+})
+
+var _ = Describe("GetCloudStorageReportURL", func() {
+	var (
+		provider      string
+		bucket        string
+		gatlingName   string
+		subDir        string
+		expectedValue string
+	)
+	BeforeEach(func() {
+		provider = "aws"
+		bucket = "testBucket"
+		gatlingName = "testGatling"
+		subDir = "subDir"
+		expectedValue = "https://testBucket.s3.amazonaws.com/testGatling/subDir/index.html"
+	})
+	Context("Provider is aws", func() {
+		It("path is aws s3 bucket", func() {
+			csp := &AWSCloudStorageProvider{providerName: provider}
+			Expect(csp.GetCloudStorageReportURL(bucket, gatlingName, subDir)).To(Equal(expectedValue))
+		})
+	})
+})
+
+var _ = Describe("GetGatlingTransferResultCommand", func() {
+	var (
+		provider             string
+		resultsDirectoryPath string
+		region               string
+		storagePath          string
+		expectedValue        string
+	)
+	BeforeEach(func() {
+		provider = "aws"
+		resultsDirectoryPath = "testResultsDirectoryPath"
+		region = "ap-northeast-1"
+		storagePath = "testStoragePath"
+		expectedValue = `
+RESULTS_DIR_PATH=testResultsDirectoryPath
+rclone config create s3 s3 env_auth=true region ap-northeast-1
+for source in $(find ${RESULTS_DIR_PATH} -type f -name *.log)
+do
+	rclone copyto ${source} --s3-no-check-bucket --s3-env-auth testStoragePath/${HOSTNAME}.log
+done
+`
+	})
+	Context("Provider is aws", func() {
+		It("provider is aws", func() {
```

Comment:
@yokawasa 
【nits】
Should test case name be like below? This seems true to other test cases in aws_test.go and gcp_test.go
```suggestion
		It("returns commands with s3 rclone config", func() {
```

Reply:
@kayamin thanks for the suggestion! I took it